owncloudApp/src/main/java/com/owncloud/android/utils/PermissionUtil.java

```diff
@@ -45,7 +45,7 @@ public static boolean shouldShowRequestPermissionRationale(Activity activity, St
      *
      * @param activity The target activity.
      */
-    public static void requestWriteExternalStoreagePermission(Activity activity) {
+    public static void requestWriteExternalStoragePermission(Activity activity) {
```

Comment:
api break

Reply:
It should not break anything if you renamed it everywhere as you did 👍